Mlem/App/Views/Root/Tabs/Settings/FiltersSettingsView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  FiltersSettingsView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-12-22.
+//
+
+import SwiftUI
+import Dependencies
+
+struct FiltersSettingsView: View {
+    @Dependency(\.persistenceRepository) var persistenceRepository
+    
+    @Environment(Palette.self) var palette
+    @Environment(FiltersTracker.self) var filtersTracker
+    
+    @State var filteredKeywords: [String]
+    @State var newKeyword: String = ""
+    
+    init() {
+        @Dependency(\.persistenceRepository) var persistenceRepository
+        
+        self._filteredKeywords = .init(wrappedValue: .init(persistenceRepository.loadFilteredKeywords()))
+    }
+    
+    var body: some View {
+        List {
+            Section {
+                keywordSection
+            } header: {
+                Text("Keyword Filters")
+            } footer: {
+                // swiftlint:disable:next line_length
+                Text("Posts containing these keywords will not be shown. If you are a moderator or administrator for a given post, it will be present in your feed but require a tap to display the content.")
```

Comment:
Wording suggestion:

```suggestion
                Text("Posts with these keywords in their titles will be hidden. If you are a moderator or administrator of a matching post, it will appear in your feed but require you to tap to view its content.")
```